documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -500,6 +500,86 @@ The generated code of the `updateCarFromDto()` method will update the passed `Ca
 
 Collection- or map-typed properties of the target bean to be updated will be cleared and then populated with the values from the corresponding source collection or map.
 
+[[direct-field-mappings]]
+=== Mappings with direct field access
+
+MapStruct also supports mappings of `public` fields that have no getters/setters. MapStruct will
```

Comment:
No it doesn't it only accepts `public` visible accessors. I think that we agreed adding the ability to use `package`/`protected` accessors as part of #572?